fire/SFMainMod.F90

```diff
@@ -916,21 +923,49 @@ subroutine  crown_damage ( currentSite )
 
           do while(associated(currentCohort))  
              currentCohort%fraction_crown_burned = 0.0_r8
+             currentCohort%passive_crown_FI      = 0.0_r8  !critical fire intensity for passive crown fire
+             currentCohort%crown_fire_flg        = 0       !flag for passvie crown fire ignition
+             
              if (EDPftvarcon_inst%woody(currentCohort%pft) == 1) then !trees only
+                
+                ! Calculate clear branch bole height at base of crown
+                ! inst%crown = crown_depth_frac (PFT)
+                height_cbb = currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft)
```

Comment:
I am rethinking this one... To represent crown base height I think you need:
`crown_height - crown_depth`
which can be written as
`crown_height * (1 - crown_depth_frac)`

```suggestion
                height_cbb = currentCohort%hite * (1._r8 - EDPftvarcon_inst%crown(currentCohort%pft))
```

Reply:
So I think that your statement is calculating crown_depth rather than height_cbb:
`height_cbb = currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft)`
 This series of suggestions that I've posted assumes that by crown base height we mean the height of the bottom of the canopy. I hope I have not misunderstood.